ios-target/Info.plist

```diff
@@ -37,10 +39,10 @@
 	<string>1</string>
 	<key>ConfigurationName</key>
 	<string>Debug</string>
+	<key>FacebookAppID</key>
+	<string>344767479509537</string>
 	<key>FacebookDisplayName</key>
-	<string></string>
-	<key>FacebookKey</key>
-	<string></string>
+	<string>Sol's first app</string>
```

Comment:
🐣

Reply:
🐀

Reply:
it was the name of the fb app by default 😐